simplecpp.cpp

```diff
@@ -1737,16 +1737,10 @@ namespace {
         }
     }
 
+    const char * const altopData[] = {"and","or","bitand","bitor","not","not_eq","xor"};
+    const std::set<std::string> altop(&altopData[0], &altopData[7]);
```

Comment:
Yes C++11 has better features. But this is a library. If somebody wants to use it I want it to be easy to use. If the compiler does not handle C++11 then it is not easy to use.